Scripts/script-TaniumMultiHash.yml

```
@@ -0,0 +1,184 @@
+      cpu_col = "Computer Name;Computer ID"
+      file_cols = "Index Query File Details{{"+param_hash+"={0},limit=1}}"
+      #file_cols = "Index Query File Details{{"+param_hash+"={0}}}"
+      question_filters = "Index Query File Exists{{"+param_hash+"={0}}},that equals:Yes"
+      all_file_cols = [file_cols.format(md5) for md5 in page]
+      all_cols = cpu_col+";"+";".join(all_file_cols)
+      all_filters=[question_filters.format(md5) for md5 in page]
+      all_filters_string=";".join(all_filters)
+
+      #demisto.info("all:"+all_cols)
+      tn_manual={}
+      tn_manual["sensors"]=all_cols
+      tn_manual["question_filters"]=all_filters_string
+      tn_manual["complete_pct"]=complete_pct
+      tn_manual["question_options"]="or"
+      hashes = {"hashes":page}
+      demisto.results([{"Type": entryTypes["note"],
+                  "ContentsFormat": formats["json"],
+                  "Contents": hashes,
+                  "HumanReadable":tableToMarkdown("Searching hashes:",hashes)
+
+              }])
+      demisto.info("demisto page hashes: " + str(page))
+      #demisto.log("tanium-question: "+str(tn_manual))
+      demisto.info("tanium-question: "+str(tn_manual))
+      res = demisto.executeCommand("tn-ask-manual-question",tn_manual)
+      #demisto.info("res:" + str(res))
+      #for debugging xml response
+      #demisto.log(res)
+      #sys.exit()
+      #error check
+      if res is None:
+          demisto.info("error res is None")
+          demisto.results([{"Type": entryTypes["error"], "ContentsFormat": formats["text"],
+                              "Contents": "Result from tanium question is None; Error"}])
+          continue
+      result = res[0]
+      if isError(result):
+          demisto.info("is error")
+          demisto.results(res)
+          continue
+      demisto.info("not error")
+
+      contents = result["Contents"]
+      human_readable = result["HumanReadable"]
+      #contents exist check
+      if not contents:
```

Not sure why are we doing all those checks if you send it to demisto.results anyway? Perhaps you should raise an exception if you have an error?

---

Raising an exception does not return an error to our war room that i can see. This may have been fixed in a newer version of Demisto though.